parcels/particlesets/baseparticleset.py

```diff
@@ -271,3 +246,257 @@ def ParticleFile(self, *args, **kwargs):
         """Wrapper method to initialise a :class:`parcels.particlefile.ParticleFile`
         object from the ParticleSet"""
         pass
+
+    def _set_particle_vector(self, name, value, indices=None):
```

Comment:
still not really generalisable - I'm gonna make a counter-proposal here.

Reply:
Same for this one. It will either be removed or rewritten.